MetaMask/metamask-extension
# Switch Flask horizontal logos

The Flask horizontal logos for "light" and "dark" mode were accidentally switched. They have now been switched back. The image `metamask-logo-horizontal.svg` is for light mode, and the image `metamask-logo-horizontal-dark.svg` is for dark mode.

Manual testing steps:
 - See the "rich diff" of this PR for each image
 - Create a Flask build and ensure the horizontal logo uses dark text (it can be seen in the fullscreen view).
